handler_models.go

```diff
@@ -235,20 +235,20 @@ func ModelOwnerVersionZip(owner, name string, user *users.User, tx *gorm.DB,
 	if !valid {
 		modelVersion = ""
 	}
-
-	model, zipPath, ver, em := (&models.Service{}).DownloadZip(r.Context(), tx,
+	svc := &models.Service{Storage: globals.CloudStorage}
+	_, zipPath, ver, em := svc.DownloadZip(r.Context(), tx,
 		owner, name, modelVersion, user, r.UserAgent())
 	if em != nil {
 		return nil, em
 	}
 
-	zipFileName := fmt.Sprintf("model-%sv%d.zip", *model.UUID, ver)
+	//zipFileName := fmt.Sprintf("model-%sv%d.zip", *model.UUID, ver)
 
 	// Remove request header to always serve fresh
-	r.Header.Del("If-Modified-Since")
+	//r.Header.Del("If-Modified-Since")
 	// Set zip response headers
 	w.Header().Set("Content-Type", "application/zip")
-	w.Header().Set("Content-Disposition", fmt.Sprintf("attachment; filename=\"%s\"", zipFileName))
+	//w.Header().Set("Content-Disposition", fmt.Sprintf("attachment; filename=\"%s\"", zipFileName))
```

Comment:
Why is this being removed?

Reply:
I assume that the S3 server handles all the HTTP headers, I kept them as comments so we can discuss it in the PR.